### ISSUE TITLE
ATM90E32 not communicating ESP32C3

### ISSUE THREAD

**Issue by jrmurdoch** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#134]
I'm getting a not communicating error, this has been setup with esphome. The device is a ESP32C3 these use the IDF framework instead of Arduino. Can anyone confirm this will work with esphome set to idf framework?

**Comment by CircuitSetup** [comment]
I dont believe the c3 is the same pinout as the regular esp32. Does it have 19 pins per side?

**Comment by jrmurdoch** [comment]
The pin out is different it's a atthinker module it's narrower so it doesn't fit so I've hooked it up with jumper wires. I've tested lots of wire combinations and double checking etc. So that why I'm wondering if it's the framework

**Comment by CircuitSetup** [comment]
were you able to get this working with the latest version of ESPHome?

**Comment by jrmurdoch** [comment]
Yes, I got this working with the C3 module.
